srcpkgs/bitwarden/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'bitwarden'
+pkgname=bitwarden
+version=1.29.1
+revision=1
+archs="x86_64"
+depends="nss libsecret libnotify libXtst libXScrnSaver"
+short_desc="Desktop app for Bitwarden password manager"
+maintainer="AI-nsley69 <trainsley69@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://bitwarden.com/"
+distfiles="https://github.com/bitwarden/desktop/releases/download/v${version}/Bitwarden-${version}-amd64.deb"
```

Comment:
why use the x86_64-only deb? Is it not possible to build from source?

Reply:
I couldn't figure out how to build it from source, the instructions seems like it'll only compile typescript to javascript, then run it through `npm run electron`.

Reply:
looks like they have a [build workflow](https://github.com/bitwarden/desktop/blob/master/.github/workflows/build.yml) for github ci. maybe that'll shed some light on it

Reply:
I'll give it a shot!